mp/src/game/shared/neo/neo_predicted_viewmodel.cpp

```diff
@@ -140,9 +140,8 @@ void CNEOPredictedViewModel::DrawRenderToTextureDebugInfo(IClientRenderable* pRe
 #endif
 
 ConVar neo_lean_debug_draw_hull("neo_lean_debug_draw_hull", "0", FCVAR_CHEAT | FCVAR_REPLICATED);
-ConVar neo_lean_speed("neo_lean_speed", "0.333", FCVAR_REPLICATED | FCVAR_CHEAT, "Lean speed scale", true, 0.0, false, 1000.0);
-// Original Neotokyo with the latest leftlean fix uses 7 for leftlean and 15 for rightlean yaw slide.
-ConVar neo_lean_yaw_peek_left_amount("neo_lean_yaw_peek_left_amount", "7.0", FCVAR_REPLICATED | FCVAR_CHEAT, "How far sideways will a full left lean view reach.", true, 0.001f, false, 0);
+ConVar neo_lean_speed("neo_lean_speed", "0.2", FCVAR_REPLICATED | FCVAR_CHEAT, "Lean speed scale", true, 0.0, false, 1000.0);
```

Comment:
I think you just made the lean speed slower here. A speed of 0.4 is one frame away from the lean coming to a full stop at the same time as in the original when recorded at host_timescale 0.1

Reply:
![image](https://github.com/user-attachments/assets/e2ea0119-b898-4e70-810f-438f94ab9eb6)

Reply:
since its using lerp we're actually very close to the end of the lean a few frames before 01:10, but 01:10 is when cl_showpos shows no more changes to our position

Reply:
keep in mind this was recorded with neo_lean_yaw_peek_left_amount 7.5, but im fairly confident that's the correct value, I explained my reasoning earlier

Reply:
damn im playing this back at host_timescale 1 and its so quick now ;(